packages/forms-web-app/src/controllers/final-comment/final-comment-submitted.js

```diff
@@ -0,0 +1,5 @@
+const { VIEW } = require('../../lib/views');
+
+exports.getFinalCommentSubmitted = async (_, res) => {
+	res.render(VIEW.FINAL_COMMENT.FINAL_COMMENT_SUBMITTED);
```

Comment:
do you need to send the case reference to the view? I believe the relevant part of the final comment object should be `finalComment.caseReference`

Reply:
apologies Yan - following discussion with Alex, the reference should be `finalComment.horizonId`

Reply:
(and it's in the session, so `req.session.finalComment.horizonId`)

Reply:
done